docs/source/archive/gsoc/reports/2025/scancodeio_varsha.rst

```diff
@@ -0,0 +1,115 @@
+###################################################
+ Adding ability to store/query downloaded packages
+###################################################
+
+**Organization:** `AboutCode <https://aboutcode.org>`_
+
+**Project:** `ScanCode.io
+<https://github.com/aboutcode-org/scancode.io>`_
+
+|  **Varsha U N**
+|  GitHub: `VarshaUN <https://github.com/VarshaUN>`_
+|  LinkedIn: `Varsha U N <https://www.linkedin.com/in/varsha-un/>`_
+
+**Mentors:**
+
+-  `Philippe Ombredanne <https://github.com/pombredanne>`_
+-  `Ayan Sinha Mahapatra <https://github.com/AyanSinhaMahapatra>`_
+
+**********
+ Overview
+**********
+
+Currently ScanCode.io scans the packages but doesn’t store it. This
```

Comment:
This is not true, scancode.io does store the packages it scans on disk, but the issue is:

- we don't have an index of stored packages
- stored per project, not centrally
- same package can be stored multiple times
- scan inputs can be deleted sometimes to save disk space, but no system to backup these packages